bin/attacks/bruteforce/bf_attack.py

```diff
@@ -1,9 +1,9 @@
 from __future__ import print_function
 
-import itertools
 import os
 
 from bin.verify_hashes.verify import verify_hash_type
+from bin.generators import word_generator
```

Comment:
Bin, because it's where all the attacks are stored. So it's where everything is coming from, bin is we're executables are stored (where everything comes from)